generators/newwebapp/index.js

```diff
@@ -312,7 +313,7 @@ module.exports = class extends Generator {
                 const isUnneededXsApp =
                     sTarget.includes("xs-app") &&
                     !(
-                        this.options.oneTimeConfig.platform === "SAP Launchpad service" ||
+                        platformIsLaunchpad ||
                         this.options.oneTimeConfig.platform === "SAP HTML5 Application Repository service for SAP BTP"
```

Comment:
Sorry, @nicogeburek - but maybe you can make this also a const:

`this.options.oneTimeConfig.platform === "SAP HTML5 Application Repository service for SAP BTP"`

Reply:
Sure, no problem. I hope I caught them all this time 😄